etc/viam-csi-pi-arm64.yml

```diff
@@ -0,0 +1,55 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
+version: 1
+script:
+- rm -rf $TARGET_APPDIR | true
+- mkdir -p "$TARGET_APPDIR/usr/bin"
+- cp ../build/viam-csi "$TARGET_APPDIR/usr/bin/viam-csi"
+- mkdir -p "$TARGET_APPDIR/usr/share/icons/viam/256x256/apps/"
+- cp ./viam-server.png "$TARGET_APPDIR/usr/share/icons/viam/256x256/apps/viam-server.png"
+- chmod 755 "$TARGET_APPDIR/usr/bin/"*
+
+AppDir:
+  path: ./AppDir
+  app_info:
+    id: com.viam.csi-module
+    name: ${PACK_NAME}
```

Comment:
Where is this and `PACK_TAG` below defined?

Reply:
These are defined in the Makefile.